namespaces/live.cloud-platform.service.justice.gov.uk/techdebt-dev/resources/ecr.tf

```diff
@@ -0,0 +1,95 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=4.9"
```

Comment:
Please bump this to version `?ref=5.0.0`

Reply:
updated to latest version ,